.doc_gen/cross-content/eventbridge_metadata.yaml

```diff
@@ -0,0 +1,182 @@
+# zexi 0.4.0

```

Comment:
This file seems to have been duplicated into the cross-content folder. Delete the cross-content version and keep the one in metadata.